i18n/en-US.properties

```diff
@@ -1,3 +1,5 @@
+# Title for the Activity Feed tab
+be.activityFeedTitle = Activity
```

Comment:
This was removed, did you rebase?

Reply:
its on line 214

Reply:
I did rebase. It's still in `messages.js`, in master
https://github.com/box/box-ui-elements/blob/master/src/components/messages.js#L333